cmd/node/config/p2p.toml

```diff
@@ -55,6 +55,10 @@
     #RoutingTableRefreshIntervalInSec defines how many seconds should pass between 2 kad routing table auto refresh calls
     RoutingTableRefreshIntervalInSec = 300
 
+    Type = "kad dht with lists"
+
 [Sharding]
     # Number of ptiority bits used
     PrioBits = 4
+    MaxIntraShard = 15
```

Comment:
15 +15 < 32 as you have TargetPeerCount. Is the rest of allocation random?

Reply:
it is the reserve for the unknown peers (such as newly connected peers that did not advertise their shard association or seeders)